template/page.phtml

```diff
@@ -0,0 +1,14 @@
+<?php require_once __DIR__ . '/../functions/variables.php'; ?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <?php require_once __DIR__ . '/../template/head.phtml'; ?>
+</head>
+<body>
+<?php require_once __DIR__ . '/../template/menu.phtml'; ?>
+<main id="<?php echo $content; ?>">
+    <?php require_once __DIR__ . '/../template/content/home.phtml'; ?>

```

Comment:
@ReginaBorisova 
You may need to use variable to load the template dynamically.  Example:
```
 <?php
  $content = 'about'; 
  require_once __DIR__ . '/../template/content/'.$content.'.phtml'; 
?>
```